Subject: [PATCH] refresh
---
Index: src/base/bittorrent/sessionimpl.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/base/bittorrent/sessionimpl.cpp b/src/base/bittorrent/sessionimpl.cpp
--- a/src/base/bittorrent/sessionimpl.cpp	(revision 733303569712db10fd349c5f5c8af90bf9f23b8e)
+++ b/src/base/bittorrent/sessionimpl.cpp	(date 1732004958236)
@@ -121,7 +121,7 @@
 namespace
 {
     const char PEER_ID[] = "qB";
-    const auto USER_AGENT = QStringLiteral("qBittorrent Enhanced/" QBT_VERSION_2);
+    const auto USER_AGENT = QStringLiteral("qBittorrent/" QBT_VERSION_2);
     const QString DEFAULT_DHT_BOOTSTRAP_NODES = u"dht.libtorrent.org:25401, dht.transmissionbt.com:6881, router.bittorrent.com:6881, router.utorrent.com:6881, dht.aelitis.com:6881"_s;

     void torrentQueuePositionUp(const lt::torrent_handle &handle)
